CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# v2.3.0
+
+This release migrates to the new `osv.dev` and `osv-schema` proto bindings for its internal data models ([#2328](https://github.com/google/osv-scanner/pull/2328)). This is primarily an internal change and should not impact users.
+
+### Features:
+
+- [Feature #2321](https://github.com/google/osv-scanner/pull/2321) Add support for license checks for RubyGems.
+- [Feature #2294](https://github.com/google/osv-scanner/pull/2294) Replace `requirementsenhanceable` extractor with transitive enricher.
+- [Feature #1417](https://github.com/google/osv-scanner/issues/1417) Add `packagedeprecation` enricher to check for deprecated packages. (`--experimental-plugins=packagedeprecation`)
```

Comment:
I don't think this packagedeprecation result being diplayed in the osv-scanner output anywhere?
@another-rex @Ly-Joey ?

Reply:
Made: https://github.com/google/osv-scanner/issues/2354

Reply:
Oh, thought it was just a feature request in osv-scalibr, not scanner. Ah well.